### PR TITLE
[#SCMS-94] - add *.map files to gitignore, update locked packages, update README with commands

### PR DESCRIPTION
**Pull request by lukaszjagodzinski**

### CODE REVIEW

**Review by markmiscavage — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -41,17 +41,35 @@ Once Pillow is installed, Scarlet can be installed via *pip*.
...
   nvm use
 ```
 
-Then, simply run the following : 
+Then, simply run the following :
 
 ```
   npm install
+```
+Which install all necessary packages for front-end part.
```
Comment (deserat) [inline README.md]: installs

File: README.md [hunk 2]
```diff
@@ -41,17 +41,35 @@ Once Pillow is installed, Scarlet can be installed via *pip*.
...
+
+For usage as static generated css/js run:
+```
   npm run build
 ```
 
-At the moment, the start task doesn't work at the moment. 
+For make changes and generate css/js files after save run:
```
Comment (deserat) [inline README.md]: To make changes

File: README.md [hunk 3]
```diff
@@ -41,17 +41,35 @@ Once Pillow is installed, Scarlet can be installed via *pip*.
...
+```
+  npm run start
+```
+Before push to PR run:
```
Comment (deserat) [inline README.md]: Before you push to PR

File: README.md [hunk 4]
```diff
@@ -41,17 +41,35 @@ Once Pillow is installed, Scarlet can be installed via *pip*.
...
+```
+  npm run buildProd
+```
+
+For test and linter run:
```
Comment (deserat) [inline README.md]: To test and lint

**Review by deserat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lukaszjagodzinski) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (lukaszjagodzinski) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (lukaszjagodzinski) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (lukaszjagodzinski) [reply README.md]: done

**Review by markmiscavage — APPROVED** [review]

**Pull request merged**